### ISSUE TITLE
Simple script does not terminate

### ISSUE THREAD

**Issue by mrcoles** [facundoolano/app-store-scraper#15]
Node v4.2.2  
app-store-scraper@0.5.0

It appears that this library prevents a script from terminating. Here’s a super simple script that when I run from the command-line as `node test.js` it just hangs until I forcefully close it.

```js
var store = require('app-store-scraper');

store.search({
    term: 'panda',
    num: 2,
    device: store.device.IOS,
    country : 'us'
})
.then(console.log)
.catch(console.log);
```

As a sanity check, the following does terminate on its own.

```js
var store = require('app-store-scraper');

function foo() {
    return new Promise((resolve, reject) => {
	resolve('OK!');
    });
}

foo()
.then(console.log)
.catch(console.log);
```

**Comment by mrcoles** [comment]
Also tested in Node v7.4.0

**Comment by facundoolano** [comment]
same as https://github.com/facundoolano/google-play-scraper/issues/137 has to do with using memoization.

**Comment by chesles** [comment]
I'm also running into this - looks like the `memoizee` package uses `setTimeout` to clear cache entries after the specified max-age, so scripts should terminate after that timer fires, it will just take [12 hours](https://github.com/facundoolano/app-store-scraper/blob/master/lib/common.js#L66) :)

**Comment by rchatterjee** [comment]
I think the issue is with `store.device.IOS` being `undefined`. Try without that and it will work.
See this [recent commit](https://github.com/facundoolano/app-store-scraper/commit/791cda966ca6cf3b742f9ab13e179382e4016581).

**Comment by facundoolano** [comment]
This should be fixed in v0.7.0 with memoization disabled by default. Please reopen if the problem persists.
